pkg/cmd/extension/command.go

```diff
@@ -220,9 +228,83 @@ func checkValidExtension(rootCmd *cobra.Command, m extensions.ExtensionManager,
 	return nil
 }
 
+func isBinExtension(client *http.Client, repo ghrepo.Interface) (isBin bool, err error) {
```

Comment:
Since this is used by the manager do you think the code should live in `manager.go`?